frontend/src/scenes/surveys/SurveySettings.tsx

```diff
@@ -38,6 +38,8 @@ function SurveyPopupToggle(): JSX.Element {
                 labelClassName="text-base font-semibold"
                 checked={!!currentTeam?.surveys_opt_in}
                 className="p-0"
+                disabled={currentTeamLoading}
```

Comment:
do you need to do this [here](https://github.com/PostHog/posthog/blob/511cbdc2a872c605e034216166799f5e61438c6a/frontend/src/scenes/surveys/surveysLogic.tsx#L343) as well?